modules/virt-binding-devices-vfio-driver.adoc

```diff
@@ -0,0 +1,132 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-configuring-pci-passthrough.adoc
+
+[id="virt-binding-devices-vfio-driver_{context}"]
+= Binding a PCI device to the VFIO driver
+
+To bind a PCI device to the VFIO driver, find the values for `vendor-ID` and `device-ID` and add them to the `/etc/modprobe.d/vfio.conf` file. Then, configure the `MachineConfig` object file for the VFIO driver and load the new driver.
+
+.Prerequisites
+. The Intel or AMD hardware must be supported by the CPU and enabled in the BIOS settings.
```

Comment:
The prerequisites are the Virtualization and IOMMU extensions that should be enabled in the BIOS and the kernel arguments that we are adding another example. I wonder if we should mention this here are again?

Reply:
We have clearly indicated in the very first procedure (Adding kernel arguments to enable IOMMU for the BIOS) these 3 prerequisites:

. Administrative privilege to a working OpenShift Container Platform cluster.
. The CPU for the device hardware belongs to either Intel or AMD brand.
. Enable the hardware virtualization extensions for Intel Virtualization Technology (Intel VT-X). AMD-V extensions are typically enabled by default.

And the intent is for the user to sequentially execute each procedure one-by-one. So my thought is that we shouldn't have to mention the prerequisite again. But if you really think it's relevant to repeat the prerequisites, let me know!